rules/.gherkin-lintrc

```diff
@@ -11,7 +11,7 @@
   "no-multiple-empty-lines": "on",
   "no-empty-file": "on",
   "no-scenario-outlines-without-examples": "on",
-  "name-length": ["on", {"Feature": 50}],
+  "name-length": ["on", {"Feature": 50, "Scenario": 140}],
```

Comment:
while we're in here can you do `Step` as well at 140 - thats the one that tripped me up.

Reply:
Done 👍